pkg/config/config.go

```diff
@@ -356,14 +359,51 @@ func performEnvVarExpansion(buf []byte, expandEnvVars bool) ([]byte, error) {
 // applied to the file and must be done manually if LoadBytes
 // is called directly.
 func LoadBytes(buf []byte, expandEnvVars bool, c *Config) error {
-	expandedBuf, err := performEnvVarExpansion(buf, expandEnvVars)
+	utf8Buf, err := EnsureUTF8(buf)
```

Comment:
nit: maybe this should be in performEnvVarExpansion so the code at this level doesn't change

Reply:
Sounds reasonable.